free-3.md

```diff
@@ -10,41 +10,104 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need met by my idea is a need for a simple beat creation for many music genres.
```

Comment:
Did you go to Colloquium a few weeks ago (October 6th, I believe) about Euterpia, the DSL for music generation? That could be a useful resource.